diff --git a/setup.py b/setup.py
index 88c2d3a..07c8518 100644
--- a/setup.py
+++ b/setup.py
@@ -364,6 +364,7 @@ class PyBuildExt(build_ext):
         lib_dirs = self.compiler.library_dirs + [
             '/lib64', '/usr/lib64',
             '/lib', '/usr/lib',
+	    '/usr/lib/x86_64-linux-gnu', '/usr/lib/i386-linux-gnu',
             ]
         inc_dirs = self.compiler.include_dirs + ['/usr/include']
         exts = []
